packages/big-design/src/components/Worksheet/Cell/styled.tsx

```diff
@@ -44,6 +44,10 @@ export const StyledCell = styled.td<StyledCellProps<WorksheetItem>>`
     css`
       background-color: ${({ theme }) => theme.colors.warning10};
     `}
+
+  & p {
+    display: block;
+  }
```

Comment:
I'm 100% sure changing line-height doesn't work as desired. Am I 100% sure something else is making my line-height not to work? No. I tried a bunch of things and finally `display` made it work.

Reply:
Ah yeah, after looking, it seems like it's a little bit of a problem. Definitely not for this PR, but a few solutions popped into my mind:
1. Add the `display` prop for typography components. That way we can do `<Small display="block" />`.
2. Allow `as="div"` for a `<Small />` or `<Text />`. I don't like this approach since it breaks a11y in many cases.

I'm good with this though 👍